From 02408c1ed0aa0be97c827e642340b6d7f2a31cab Mon Sep 17 00:00:00 2001
From: Bryce Harrington <bryce@bryceharrington.org>
Date: Thu, 12 Feb 2009 21:28:45 -0800
Subject: [PATCH] ru: Fix shift key for Russian letters; include -/_ in Russian Phonetic layout

When entering capital Russian letters using Shift key, it prints lower
case symbol.  To fix, change "key <LatQ>..." to "key <AD01>...", etc.

-/_ is missing.  Add "key <AE11>..." for this.

Signed-off-by: Bryce Harrington <bryce@bryceharrington.org>
---
 symbols/ru |   53 +++++++++++++++++++++++++++--------------------------
 1 files changed, 27 insertions(+), 26 deletions(-)

diff --git a/symbols/ru b/symbols/ru
index dda7281..a6fa0e6 100644
--- a/symbols/ru
+++ b/symbols/ru
@@ -153,34 +153,35 @@ xkb_symbols "phonetic" {
     key	<AC10> {	[	semicolon,	     colon	]	};
     key	<AC11> {	[      apostrophe,	  quotedbl	]	};
     key	<LSGT> {	[	      bar,	 brokenbar	]	};
+    key	<AE11> {	[	    minus,	underscore	]	};
 
     key	<TLDE> {	[     Cyrillic_yu,     Cyrillic_YU	]	};
-    key	<LatQ> {	[     Cyrillic_ya,     Cyrillic_YA	]	};
-    key	<LatZ> {	[     Cyrillic_ze,     Cyrillic_ZE	]	};
-    key	<LatS> {	[     Cyrillic_es,     Cyrillic_ES	]	};
-    key	<LatA> {	[      Cyrillic_a,	Cyrillic_A	]	};
-    key	<LatW> {	[     Cyrillic_ve,     Cyrillic_VE	]	};
-    key	<LatC> {	[    Cyrillic_tse,    Cyrillic_TSE	]	};
-    key	<LatX> {	[Cyrillic_softsign,Cyrillic_SOFTSIGN	]	};
-    key	<LatD> {	[     Cyrillic_de,     Cyrillic_DE	]	};
-    key	<LatE> {	[     Cyrillic_ie,     Cyrillic_IE	]	};
-    key	<LatV> {	[    Cyrillic_zhe,    Cyrillic_ZHE	]	};
-    key	<LatF> {	[     Cyrillic_ef,     Cyrillic_EF	]	};
-    key	<LatT> {	[     Cyrillic_te,     Cyrillic_TE	]	};
-    key	<LatR> {	[     Cyrillic_er,     Cyrillic_ER	]	};
-    key	<LatN> {	[     Cyrillic_en,     Cyrillic_EN	]	};
-    key	<LatB> {	[     Cyrillic_be,     Cyrillic_BE	]	};
-    key	<LatH> {	[     Cyrillic_ha,     Cyrillic_HA	]	};
-    key	<LatG> {	[    Cyrillic_ghe,    Cyrillic_GHE	]	};
-    key	<LatY> {	[   Cyrillic_yeru,   Cyrillic_YERU	]	};
-    key	<LatM> {	[     Cyrillic_em,     Cyrillic_EM	]	};
-    key	<LatJ> {	[ Cyrillic_shorti, Cyrillic_SHORTI	]	};
-    key	<LatU> {	[      Cyrillic_u,	Cyrillic_U	]	};
-    key	<LatK> {	[     Cyrillic_ka,     Cyrillic_KA	]	};
-    key	<LatI> {	[      Cyrillic_i,	Cyrillic_I	]	};
-    key	<LatO> {	[      Cyrillic_o,	Cyrillic_O	]	};
-    key	<LatL> {	[     Cyrillic_el,     Cyrillic_EL	]	};
-    key	<LatP> {	[     Cyrillic_pe,     Cyrillic_PE	]	};
+    key	<AD01> {	[     Cyrillic_ya,     Cyrillic_YA	]	};
+    key	<AB01> {	[     Cyrillic_ze,     Cyrillic_ZE	]	};
+    key	<AC02> {	[     Cyrillic_es,     Cyrillic_ES	]	};
+    key	<AC01> {	[      Cyrillic_a,	Cyrillic_A	]	};
+    key	<AD02> {	[     Cyrillic_ve,     Cyrillic_VE	]	};
+    key	<AB03> {	[    Cyrillic_tse,    Cyrillic_TSE	]	};
+    key	<AB02> {	[Cyrillic_softsign,Cyrillic_SOFTSIGN	]	};
+    key	<AC03> {	[     Cyrillic_de,     Cyrillic_DE	]	};
+    key	<AD03> {	[     Cyrillic_ie,     Cyrillic_IE	]	};
+    key	<AB04> {	[    Cyrillic_zhe,    Cyrillic_ZHE	]	};
+    key	<AC04> {	[     Cyrillic_ef,     Cyrillic_EF	]	};
+    key	<AD05> {	[     Cyrillic_te,     Cyrillic_TE	]	};
+    key	<AD04> {	[     Cyrillic_er,     Cyrillic_ER	]	};
+    key	<AB06> {	[     Cyrillic_en,     Cyrillic_EN	]	};
+    key	<AB05> {	[     Cyrillic_be,     Cyrillic_BE	]	};
+    key	<AC06> {	[     Cyrillic_ha,     Cyrillic_HA	]	};
+    key	<AC05> {	[    Cyrillic_ghe,    Cyrillic_GHE	]	};
+    key	<AD06> {	[   Cyrillic_yeru,   Cyrillic_YERU	]	};
+    key	<AB07> {	[     Cyrillic_em,     Cyrillic_EM	]	};
+    key	<AC07> {	[ Cyrillic_shorti, Cyrillic_SHORTI	]	};
+    key	<AD07> {	[      Cyrillic_u,	Cyrillic_U	]	};
+    key	<AC08> {	[     Cyrillic_ka,     Cyrillic_KA	]	};
+    key	<AD08> {	[      Cyrillic_i,	Cyrillic_I	]	};
+    key	<AD09> {	[      Cyrillic_o,	Cyrillic_O	]	};
+    key	<AC09> {	[     Cyrillic_el,     Cyrillic_EL	]	};
+    key	<AD10> {	[     Cyrillic_pe,     Cyrillic_PE	]	};
 
     key	<AD11> {	[    Cyrillic_sha,    Cyrillic_SHA	]	};
     key	<AE12> {	[    Cyrillic_che,    Cyrillic_CHE	]	};
-- 
1.6.0.4

